module/VuFind/src/VuFind/Search/Base/Options.php

```diff
@@ -591,11 +590,11 @@ public function getHandlerForLabel($label)
      * Given a basic handler name, return the corresponding label (or false
      * if none found):
      *
-     * @param string $handler Handler name to look up.
+     * @param ?string $handler Handler name to look up.
      *
      * @return string
      */
-    public function getLabelForBasicHandler($handler)
+    public function getLabelForBasicHandler(?string $handler): string
```

Comment:
This appears to actually return `string|bool` based on the code below. Probably better to return `?string` instead. Seems like we must be missing a test case for this scenario since this type problem didn't get caught.

Reply:
Ahh, makes sense -- but it would fail if we switched to strict types in the file.